src/components/GameItem.js

```diff
@@ -4,23 +4,39 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  constructor(props) {
+    super(props)
+    this.state = {
+      addClickCss: ""
+    }
+  }
+  
+  onNatureItemClick = () => {
+    this.setState(
+      {
+        addClickCss: this.props.iconName === "litter" ?
```

Comment:
I think there is some room for improvement around your naming here.

`onNatureItemClick` is close to the convention, but not all of the game items are nature items (some are litter). Since we don't have any other click handlers to worry about, I might shorten the name of this to just `onClick`.

`addClickCss` isn't a great name for a piece of state. The word "add" makes it a verb phrase, which we usually reserve for functions. Something like `overlayClass` or `clickedClass` might be a better choice.